pkg/scripts/node.go

```diff
@@ -29,6 +29,17 @@ var (
 		sudo KUBECONFIG=/etc/kubernetes/admin.conf \
 		kubectl drain {{ .NODE_NAME }} --ignore-daemonsets --delete-local-data
 	`)
+
+	restartKubeAPIServerTemplate = heredoc.Doc(`
+		apiserver_id=$(sudo crictl ps --name=kube-apiserver --q)
+		[ -z "$apiserver_id" ] && exit 1
+		
+		apiserver_logs=$(sudo crictl logs "$apiserver_id" 2>&1)
```

Comment:
This prints the whole API server logs when running KubeOne with the verbose flags. I'm not sure how to solve that, but I'm open to suggestions.

Reply:
How about — redirect it to the file to later inspection if it's needed?

`sudo crictl logs "$apiserver_id" 2>&1 > /tmp/apiserver.log`

Reply:
Done.